Doc/library/urllib.error.rst

```diff
@@ -31,14 +31,19 @@ The following exceptions are raised by :mod:`urllib.error` as appropriate:
       of :exc:`IOError`.
 
 
-.. exception:: HTTPError
+.. exception:: HTTPError(url, code, msg, hdrs, fp)
 
    Though being an exception (a subclass of :exc:`URLError`), an
    :exc:`HTTPError` can also function as a non-exceptional file-like return
    value (the same thing that :func:`~urllib.request.urlopen` returns).  This
    is useful when handling exotic HTTP errors, such as requests for
    authentication.
 
+   .. attribute:: url
+
+      Contains the request URL.
+      Is an alias for *filename* attribute.
```

Comment:
Maybe more concisely, the 'Is' can be dropped here and below? Not sure which style's better though.

Reply:
I am not a native speaker, so I am fine with any decision, which sounds better :)

Reply:
Not English native speaker either, but "Alias for ..." sounds better to me.